filebeat/input/tcp/config.go

```diff
@@ -0,0 +1,23 @@
+package tcp
+
+import (
+	"github.com/elastic/beats/filebeat/harvester"
+)
+
+type config struct {
+	harvester.ForwarderConfig `config:",inline"`
+	Host                      string `config:"host"`
+	LineDelimiter             string `config:"line_delimiter" validate:"nonzero"`
+	Timeout                   int    `config:"timeout" validate:"nonzero,positive"`
+	MaxReadBuffer             int64  `config:"max_read_buffer" validate:"nonzero,positive"`
+}
+
+var defaultConfig = config{
+	ForwarderConfig: harvester.ForwarderConfig{
+		Type: "tcp",
+	},
+	LineDelimiter: "\n",
+	Host:          "localhost:9000",
```

Comment:
Is 9000 a "common" default?

Reply:
@ruflin Not sure about that, I think it's has good as the default we have in UDP. :)

Reply:
👍